hspec-core/src/Test/Hspec/Core/Example.hs

```diff
@@ -53,12 +53,14 @@ class Example e where
 data Params = Params {
   paramsQuickCheckArgs  :: QC.Args
 , paramsSmallCheckDepth :: Maybe Int
+, paramsUseColor :: Bool
```

Comment:
I *generally* prefer to see more specific types - `Bool`ean blindness can be an issue when reading code. `data UseColor = YesUseColor | Don'tUseColor` or something analogous.

Reply:
> Looks good to me - is this just to support having configured color output in the hspec-hedgehog?

Yes, exactly. As of now I can't get colors from `hspec-hedgehog` when redirecting `stdout`.

I'm mentally toying around with a different approach though. I think technically it's the responsibility of the formatter to decide on when to use color output. I'll ping you when I get around to sketch something up.